### PR TITLE
Additions made to samples user guide

### LINKED ISSUE #57: Samples updates
- delete a sample
- files & caching
  - downloading sample files
  - what are caches
- default subtractions
  - explain
  - update create sample user guide
- quality
  - sample vs cache quality
  - interpretation (examples from James and Lan?)
- edit sample
- update settings screenshots

### PR DESCRIPTION
**Pull request by mdhaul**
- resolves #57 
- setting screenshots updated
- delete/edit a sample added
- info on caching included at the top (should this section be somewhere else on the page?)
- default subtraction added as a subsection at the end of "create a sample" section 
- quality analysis information added with interpretation

### CODE REVIEW

File: content/docs/manual/ug_samples/index.md [hunk 1]
```diff
@@ -91,17 +114,114 @@ Quality information is calculated from the library using [FastQC](https://www.bi
 
 The FASTQ files and quality data are used for further analyses triggered by the user.
 
+## Default Subtraction
```
Comment (igboyes) [inline content/docs/manual/ug_samples/index.md]: Can you add a screenshot of selecting a Default Subtraction during sample creation.

File: content/docs/manual/ug_samples/index.md [hunk 2]
```diff
@@ -71,37 +122,116 @@ For paired data, make sure the file orientation labels (left and right) are corr
...
+
+# Download a FASTQ File
+
+You can download the original FASTQ files used to create a sample...
+
+# Trim Caches
 
+Sample data is automatically trimmed during analysis to remove sequencing artefacts and low quality regions. After the first analysis, the trimmed data are cached for reuse in future analyses that use the same trimming parameters. This saves running the trimming workflow steps for every analysis for a given sample.
+
+A sample that has not yet been analyzed will not have any caches associated with it.
```
Comment (igboyes) [inline content/docs/manual/ug_samples/index.md]: Make new sample and take shot of empty Caches section.

File: content/docs/manual/ug_samples/index.md [hunk 3]
```diff
@@ -71,37 +122,116 @@ For paired data, make sure the file orientation labels (left and right) are corr
...
+
+_PICTURE_
+
+Running an analysis for this sample will create an analysis job. During the job the raw sample reads will be trimmed and cached for future analyses. As soon as trimming is complete the cache will be created.
```
Comment (igboyes) [inline content/docs/manual/ug_samples/index.md]: Take shot when cache has been added during analysis.

File: content/docs/manual/ug_samples/index.md [hunk 4]
```diff
@@ -71,37 +122,116 @@ For paired data, make sure the file orientation labels (left and right) are corr
...
+
+_PICTURE_
+
+
+
+
+
+For example, when a new sample is run using a subtraction that has already been used, the host genome that has already been analyzed and eliminated will not have to go through the same process again. Instead the reads belonging to the host genome or non-pathogenic organisms associated with the host such as insects or fungi will already be cached. This makes the analyzing process for a sample(s) happen a lot faster and decreases the amount of work the server has to perform.
```
Comment (igboyes) [inline content/docs/manual/ug_samples/index.md]: It is just the quality trimming that is cached. This paragraph should be removed.

File: content/docs/manual/ug_samples/index.md [hunk 5]
```diff
@@ -71,37 +122,116 @@ For paired data, make sure the file orientation labels (left and right) are corr
...
+
+# Cache Quality Analysis
```
Comment (igboyes) [inline content/docs/manual/ug_samples/index.md]: Sorry, I think I confused you. I'll try to explain. Analysis bit may be better in **Analyses** section of docs.

**Comment by mdhaul** [comment]
Ok so I think I've made all the changes we talked about. I've put the comparison for sample vs cache quality under "Trim Caches". There is also a separate section called "Sample Quality" under "Create Sample" that also talks about the samples quality. Is that too repetitive? The "Sample Quality" section does not include cache quality because I've added that under "Trim Catches".

Let me know your thoughts.

**Pull request merged**